src/apiCalls.js

```diff
@@ -1,7 +1,7 @@
 // @flow
-import fetch from 'isomorphic-fetch';
 import {formatQueryParams} from './utils';
 import {getUrlForContactPoint, getPublicApiUrl, getContactPoint} from './globals';
+import fetch from './fetch';
```

Comment:
Sure, I was trying to make it transparent, thus the name, but I'll make it explicit it is our version of fetch